src/main.c

```diff
@@ -612,6 +635,17 @@ static void calculate_setpoint_target(Data *d) {
             // The rider has 500ms to react to the triple-beep, or maybe it was just a short spike
             d->state.sat = SAT_NONE;
         }
+
+    } else if (bms_get_fault(d->bms_fault, BMSF_CONNECTION)) {
+        beep_alert(d, 3, true);
+        d->beep_reason = BEEP_BMS_CONNECTION;
+
+        if (d->motor.erpm > 0) {
+            d->setpoint_target = d->float_conf.tiltback_hv_angle;
+        } else {
+            d->setpoint_target = -d->float_conf.tiltback_hv_angle;
+        }
+        d->state.sat = SAT_PB_BMS_CONNECTION;
```

Comment:
I'm not sure about adding another SAT at all, but if we do, I'd call it just generic `SAT_PB_ALERT` with the intention to use it for any sort of alert besides the ones already present. But not sure this needs a pushback, more so seeing you're also doing a beeper when in `STATE_READY` - a buzz would probably suffice, but the current alerting infra is very limiting.